fatiando/gravmag/harvester.py

```diff
@@ -526,34 +533,42 @@ def _is_neighbor(index, props, neighborhood):
     return False
 
 
-def _neighbor_indexes(n, mesh):
+def _neighbor_indexes(n, mesh, restrict=None):
```

Comment:
The default values can be used only in the top-level function (`harvest` in this case) and it will be passed along to the other functions. If you put it everywhere, you'll have to remember to change all of them if you want to change the default.